sql/moz-fx-data-shared-prod/fxci_derived/worker_costs_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+friendly_name: Firefox-CI Worker Costs
+description: |-
+  Derived Firefox-CI worker cost data from the GCP billing export.
+owners:
+- ahalberstadt@mozilla.com
+labels:
+  incremental: true
+  owner1: ahalberstadt
+scheduling:
+  dag_name: bqetl_internal_tooling
+bigquery:
+  time_partitioning:
+    type: day
+    field: ''
+    require_partition_filter: true
```

Comment:
This usually helps to limit query costs. It might matter less here but it's better to keep it unless table is small and we don't use the partitioning field in queries.